### PR TITLE
use -default-jvm for all openjdk-11 dependencies

### PR DESCRIPTION
**Pull request by joshrwolf**
Fixes: 

Related:

### CODE REVIEW

File: bazel-6.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ environment:
       - python3
       - zip
       - bash
-      - openjdk-11
+      - openjdk-11-default-jvm
```
Comment (patflynn) [inline bazel-6.yaml:18]: here too

File: gradle-stage0.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
   version: 8.0.1
-  epoch: 0
+  epoch: 1
   description: A Java project management and project comprehension tool.
   copyright:
     - license: Apache-2.0
   dependencies:
     runtime:
-      - openjdk-11-jre
+      - openjdk-11-default-jvm
```
Comment (patflynn) [inline gradle-stage0.yaml:10]: meta: we need to decide whether we runtime include jvm or not.  @kaniini what happens if two packages in the dependency tree provide default-jvm but for different jdk versions?

**Comment by patflynn** [comment]
@kaniini @joshrwolf I think we want to close this out right? My understanding is that we should set JAVA_HOME for each of these packages.

**Comment by joshrwolf** [comment]
closing this in favor of `JAVA_HOME` and piece meal `-default-jvm` alternatives

**Pull request closed without merging**